Docs/beta/index.rst

```diff
@@ -0,0 +1,3 @@
+============
+Hello World!
+============
```

Comment:
LGTM. I'll build on the index doc and add as needed to the TOC, starting with the template for the how-to docs.

One comment about links: Is there a policy about 1) when to use :ref: vs. :doc: links, and 2) whether to use absolute or relative paths in on-site :doc: links? We need to make sure nothing breaks when we move the beta doc to its production path.

Reply:
That's a good point.
1) when to use :ref: vs. :doc: links
-- I don't think we have any preference on either side. I'm just considering :doc: as a special case of `:ref:`. It might be slightly more helpful for the source code readers to use `:doc:` over `:ref:` whenever possible, but I don't think this will make any difference in the resulting html file in most Sphinx themes.
2) whether to use absolute or relative paths in on-site :doc: links
-- I'd like to encourage using relative paths since all the contents in "Docs/beta" directory will probably be migrated to "Docs" some day

Reply:
I think there are some cases where you have to use one or the other -- pretty sure sphinx-design cards have a :ref: link type, but no way to link to a :doc: (unless you hard-code in the production URL, which just ... no).

IMO the :ref: links are more versatile but also more error-prone and time consuming, since you have to keep track of where they are and be careful not to duplicate them.

Re relative paths: I think absolute paths are relative to the root document you specify in the conf.py file. If that's the case, absolute paths could be a cleaner design (especially in cases where you have parent directories in the relative path). You'd only have to change the conf.py file when you switched over. I don't think it makes a lot of practical difference overall, but since we're starting from scratch it would be nice to be consistent.